.circleci/config.yml

```diff
@@ -7,10 +7,22 @@ jobs:
     steps:
       - run: sudo apt update && sudo apt install -y graphviz
       - checkout
-      - run: virtualenv venv
-      - run: source venv/bin/activate && make installdeps
-      - run: source venv/bin/activate && make lint
-      - run: source venv/bin/activate && make test
+      - run: virtualenv test_env && virtualenv dev_env
```

Comment:
can we add any of these repeated steps to the make file? under a command like `build-circle-ci`

Reply:
We can try using commands instead
https://circleci.com/docs/2.0/configuration-reference/#commands-requires-version-21